cmd/juju/commands/version.go

```diff
@@ -52,19 +52,19 @@ type versionCommand struct {
 	showAll bool
 }
 
-func newVersionCommand() *versionCommand {
-	return &versionCommand{}
+func NewVersionCommand() *VersionCommand {
```

Comment:
Why did this get exported?

Reply:
Hi @hpidcock! Thanks for pointing this out. I incorrectly thought that the command and the struct were part of a different package.

Reply:
Reverted on https://github.com/juju/juju/pull/15841/commits/72164eb2899f2a9c17f686912c50a068b507ff54.